src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,21 @@
+import { HStack, Flex } from "@chakra-ui/react"
+import { PropsWithChildren } from "react"
+
+export const Footer = ({
+  children,
+}: PropsWithChildren<Record<never, any>>): JSX.Element => (
```

Comment:
just double checking here - would this not work with `PropsWithChildren<{}>`? it's a very minor change so dw about it but curious

Reply:
I guess it does too! But I was getting a linting error, and the suggestion for empty objects was to do something like this instead